File: 03BubbleSort.py

def bubble_sort(nums):
    sorted = True
    while sorted:
        sorts = 0
        for i in range(len(nums) - 1):
            if nums[i + 1] < nums[i]:
                sorts += 1
                number = nums[i]
                nums[i] = nums[i + 1]
                nums[i + 1] = number
            else:
                sorts += 0
        sorted = bool(sorts)
    return nums

if __name__ == '__main__':
    nums = [10, 5, 3, 8, 2, 6, 4, 7, 9, 1]
    print(bubble_sort(nums))
    testing this

File: 03BubbleSort.js
function bubbleSort(nums) {
	do {
		var sort = 0;
		for (var i = 0; i < nums.length; i++) {
			if (nums[i + 1] < nums[i]) {
				var larger = nums[i];
				nums[i] = nums[i + 1];
				nums[i + 1] = larger;
				sort += 1;
			}
		}

	} while (sort)

	return nums
}

function savio() {
	testing
}

console.log(bubbleSort([10, 5, 3, 8, 2, 6, 4, 7, 9, 1]))

	savio abuga
	savio abuga
	savio abuga
